.pre-commit-config.yaml

```diff
@@ -45,7 +51,7 @@ repos:
         args: ["--profile", "black"]
 
   - repo: https://github.com/nbQA-dev/nbQA
-    rev: 1.6.4
+    rev: 1.6.3
```

Comment:
Pre-commit auto-update just bumped this to `1.6.4` (see #241). Does 1.6.4 not run cleanly?